css/_reactions-menu.scss

```diff
@@ -1,5 +1,31 @@
 @use 'sass:math';
 
+#overflow-context-menu {
```

Comment:
Can we move this to JSS? @robertpin has been moving SCSS code to JSS for a while, it would be nice not to add more.

Reply:
will give it a try